src/main/kotlin/com/reicheltp/celtic_rituals/rituals/bowl/RitualBowlBlock.kt

```diff
@@ -222,11 +240,7 @@ class RitualBowlBlock : Block(
                         val item = player.heldItemMainhand.split(1)
                         tile.setInventorySlotContents(i, item)
 
-                        return true
-                    }
-                    stack.canBeCombined(player.heldItemMainhand) -> {
-                        player.heldItemMainhand.shrink(1)
```

Comment:
Should it not be possible anymore to increase the stack of the last Item in the inventory of the ritual bowl when the item in the player hand matches the last item? Also should we not check if we can combine the placed item with any existing slot in the RitualBowlBlock?

Reply:
Yes, this is intended that you can only add exactly 5 items. No matter if they stack.